chore: add missing i18n keys in sidebar, assets, toolbox, dropdowns

christian-byrne opened the pull request:
This PR 

- adds missing locale keys for 3D viewer toast strings, assets sidebar labels, and node error keys
- cleans up the selection toolbox, media previews, node components, and widget uploader to rely on `$t`/`st` (exposed to template scope at compile time) instead of importing from `useI18n`.
- updates `eslint.config.ts` to teach the Intlify rule about the locale layout

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-6622-chore-add-missing-i18n-keys-in-sidebar-assets-toolbox-dropdowns-2a36d73d365081ae8694eb4f8ebb822a) by [Unito](https://www.unito.io)

christian-byrne committed:
chore: add missing i18n keys and tooling

DrJKL commented on eslint.config.ts:
```diff
@@ -33,7 +33,18 @@ const settings = {
+  'vue-i18n': {
+    localeDir: [
+      {
+        pattern: './src/locales/*/*.json',
```
Not sure if we'd ever add extra nesting here.

christian-byrne committed:
Update eslint.config.ts

Co-authored-by: Alexander Brown <drjkl@comfy.org>

christian-byrne committed:
Merge main into i18n/add-missing-keys

christian-byrne committed:
[feat] i18n: add Browse Files translation key to WidgetFileUpload

- Update WidgetFileUpload.vue to use $t('widgetFileUpload.browseFiles')
- Add browseFiles key to en/main.json under widgetFileUpload section
- Update unit test mock i18n to include browseFiles key
